tools/proofers/srchrep.php

```diff
@@ -131,6 +131,7 @@ function preg_quote(str)
     <input type="button" id='undo' value="<?php echo attr_safe(_("Undo.")); ?>" onClick="restore_saved_text()" disabled>
 </center>
 </form>
+<p><?php echo _("Note: \\n will be treated as a newline in the replace field."); ?></p>
```

Comment:
Am I correct that \n is available only for regex s/r? If so, this note might be better off in the "Regular Expression?" help area. Otherwise, I have a slight preference for removing the note.

Reply:
No, checking the regular expression checkbox is not required. That checkbox controls what text will get selected to be replaced, not what it will be replaced with. This change would disallow replacing with the text "\n" and always interpreting that as a newline character.

Thanks for your opinion on the note.

Reply:
> No, checking the regular expression checkbox is not required. That checkbox controls what text will get selected to be replaced, not what it will be replaced with. This change would disallow replacing with the text "\n" and always interpreting that as a newline character.
> 
Thanks for the clarification -- I was forgetting that (I'm used to `\n` being used only in regexes, both for search and replace).
> Thanks for your opinion on the note.
It's not that I don't like things to be documented -- I am just very leery about adding text to the s/r window. I'd rather keep the actual s/r form simple and slim. I notice, though, that search/replace is not documented in the prooffacehelp for either the standard or enhanced version -- I think this would be a good place to document it. We could also consider adding something to the "Proofing Interface tips and tricks" DP wiki page.

Reply:
> ... search/replace is not documented in the prooffacehelp for either the standard or enhanced version -- I think this would be a good place to document it.

There appears to be a request to add documentation for search/replace as [task 1740](https://www.pgdp.net/c/tasks.php?action=show&task_id=1740)

Reply:
I would prefer we punt documenting this into task 1740 and just do the replacement. The odds of someone wanting to do a replace with a '\n' and not mean a newline is almost nil.

Let's remove the hint para and add comment in task 1740 on what needs to be included for it in the search/replace doc update.

Reply:
Works for me.

Reply:
I updated the code to remove the note and I can take a look at 1740.